main.tf

```
@@ -26,7 +36,9 @@ locals {
   # Determine connection mode and IP ranges
   connect_mode  = var.use_private_g_services ? "PRIVATE_SERVICE_ACCESS" : "DIRECT_PEERING"
   ip_cidr_range = var.use_private_g_services ? null : var.ip_cidr_range
-
+  # Read-replica for Redis memorystore
+  replicas_mode = var.read_replicas_enabled ? "READ_REPLICAS_ENABLED" : "READ_REPLICAS_DISABLED"
+  replica_count = var.read_replicas_enabled ? var.replica_count : null
```

It's helpful to use common prefix for similarly grouped items. Helps to search words more effectively later.
```suggestion
  replica_mode = var.read_replicas_enabled ? "READ_REPLICAS_ENABLED" : "READ_REPLICAS_DISABLED"
  replica_count = var.read_replicas_enabled ? var.replica_count : null
```